packages/devextreme/js/__internal/grids/grid_core/columns_controller/m_columns_controller_utils.ts

```diff
@@ -915,7 +915,15 @@ export const mergeColumns = (that: ColumnsController, columns, commandColumns, n
   return result;
 };
 
-export const isColumnFixed = (that: ColumnsController, column) => (isDefined(column.fixed) || !column.type ? column.fixed && column.fixedPosition !== StickyPosition.Sticky : that._isColumnFixing());
+export const isColumnFixed = (that: ColumnsController, column) => {
+  const isNotCommandColumn = !column.type || isDefined(column.fixed) || column.type === AI_COLUMN_NAME;
```

Comment:
Let's try to invert the condition and omit extra check for column.fixed. Something like this:
```suggestion
const isFixedCommandColumn = column.type && column.type !== AI_COLUMN_NAME;
if(!isFixedCommandColumn) {
    return column.fixed && column.fixedPosition !== StickyPosition.Sticky;
}
```

Reply:
Did it and tested. It works, thank you!

Added fix in next commit